crowbar_framework/lib/crowbar/backup/export.rb

```diff
@@ -123,7 +123,13 @@ def crowbar
           else
             # copy files with higher permissions
             dest = data_dir.join(destination).to_s
-            system("sudo", "cp", "-a", source, dest)
+            if source == "/var/lib/crowbar"
+              # avoid doing an export of the existing backups, to avoid growing
+              # size of backups
+              system("sudo", "rsync", "-a", "#{source}/", "--exclude", "backup", dest)
+            else
+              system("sudo", "cp", "-a", source, dest)
```

Comment:
Not really, because we don't want to exclude backup subdirectories in other directories :-)